README.md

```diff
@@ -11,7 +11,6 @@ RESTMockServer.whenGET(pathContains("users/defunkt"))
             .thenReturnFile(200, "users/defunkt.json");
 ```
 **Article**
-
 [ITDD - Instrumentation TDD for Android](https://medium.com/@andrzejchm/ittd-instrumentation-ttd-for-android-4894cbb82d37)
 ##Table of Contents
 - [About](#about)
```

Comment:
we should add a link to the section about Unit tests to ToC:
`- [Unit Tests with Robolectric](#unit-tests-with-robolectric)`
just before `Android sample project`